Movie/Movie/DetailMoviePage/ActorCollectionViewCell.swift

```diff
@@ -0,0 +1,70 @@
+// ActorCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Настройка  Актеров
+final class ActorCollectionViewCell: UICollectionViewCell {
+    // MARK: Private Constant
+
+    private enum Constants {
+        static let getImageURL = "https://image.tmdb.org/t/p/w500"
+    }
+
+    // MARK: - Private Visual Components
+
+    private let actorImageView: UIImageView = {
+        let image = UIImageView(frame: CGRect(x: 5, y: 5, width: 140, height: 190))
+        return image
+    }()
+
+    private let actorNameLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 5, y: 200, width: 140, height: 20))
+        label.font = .systemFont(ofSize: 14)
+        label.textAlignment = .center
+        label.numberOfLines = 2
+        return label
+    }()
+
+    private let actorRoleLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 5, y: 220, width: 140, height: 20))
+        label.textColor = .lightGray
+        label.font = .systemFont(ofSize: 13)
+        label.textAlignment = .center
+        label.numberOfLines = 2
+        return label
+    }()
+
+    // MARK: - Init
+
+    override private init(frame: CGRect) {
+        super.init(frame: frame)
+        addSubview(actorImageView)
+        addSubview(actorNameLabel)
+        addSubview(actorRoleLabel)
+        actorImageView.layer.cornerRadius = 20
+        actorImageView.clipsToBounds = true
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Metods
+
+    func setupUI(actror: Actor) {
+        guard let urlImage = URL(string: Constants.getImageURL + actror.actorImageName) else { return }
```

Comment:
```suggestion
        guard let urlImage = URL(string:  else { return }
```
конкатенация меняем на интерполяцию везде